### PR TITLE
ENH : add new notebook about how to interpret IQMs

### PR DESCRIPTION
**Pull request by celprov**
With this notebook, we are contributing an educational resource on how to interpret and select the most important image quality metrics (IQMs) generated by MRIQC.

### CODE REVIEW

File: docs/auto-qc/iqms_intepretability.md [hunk 1]
```diff
@@ -0,0 +1,1288 @@
+# Interpretability of the Image Quality Metrics (IQMs) of MRIQC
+
+[MRIQC](https://mriqc.readthedocs.io/en/latest/) is a powerful tool to assess the quality of MR images in a research study. In addition to a visual report, a number of image quality metrics (IQMs) is generated. However, there is a large number of these metrics and it is not immediately obvious which IQM a researcher should pay most attention to when deciding over the quality of a given image.
```
Comment (oesteban) [inline docs/auto-qc/iqms_intepretability.md]: Can I ask to break lines after period (for readability):
```suggestion
[MRIQC](https://mriqc.readthedocs.io/en/latest/) is a powerful tool to assess the quality of MR images in a research study.
In addition to a visual report, a number of image quality metrics (IQMs) are generated.
However, there is a large number of these metrics and it is not immediately obvious which IQM a researcher should pay most attention to when deciding over the quality of a given image.
```

(also, fixed a typo)

File: docs/auto-qc/iqms_intepretability.md [hunk 2]
```diff
@@ -0,0 +1,1288 @@
...
+
+In this notebook, we will explore these issues in the MR-ART dataset, to provide researchers guidance in interpreting and selecting the most important IQMs from MRIQC. If you want to follow along and run the notebook yourself, please download the runnable notebook at [https://github.com/brainhack-ch/interpret-iqms/blob/main/code/interpretability-of-iqms.ipynb](https://github.com/brainhack-ch/interpret-iqms/blob/main/code/interpretability-of-iqms.ipynb).
```
Comment (oesteban) [inline docs/auto-qc/iqms_intepretability.md]: Line breaks, please correct throughout
```suggestion
In this notebook, we will explore these issues in the MR-ART dataset, to provide researchers guidance in interpreting and selecting the most important IQMs from MRIQC.
If you want to follow along and run the notebook yourself, please download the runnable notebook at [https://github.com/brainhack-ch/interpret-iqms/blob/main/code/interpretability-of-iqms.ipynb](https://github.com/brainhack-ch/interpret-iqms/blob/main/code/interpretability-of-iqms.ipynb).
```

Comment (oesteban) [reply docs/auto-qc/iqms_intepretability.md]: Please add a mention and link to your OHBM abstract (OSF link)

File: docs/auto-qc/iqms_intepretability.md [hunk 3]
```diff
@@ -0,0 +1,1288 @@
...
+        (hm2 == True)]
+    choices = [1, 2, 3]
+    scores['condition'] = np.select(conditions, choices)
+    return scores
+scores = add_condition_column(scores)
+```
+
+## Rating scores and image conditions
+
+We can explore how well the raters align in their scores with the motion condition. Does their rating reflect how much people moved in the scanner? Let's plot the confusion matrix of the scores assigned by the rater and the motion condition :
```
Comment (oesteban) [inline docs/auto-qc/iqms_intepretability.md]: Line breaks, and please do not use French semicolons (i.e., remove the space between the last word and punctuation)
```suggestion
We can explore how well the raters align in their scores with the motion condition.
Does their rating reflect how much people moved in the scanner?
Let's plot the confusion matrix of the scores assigned by the rater and the motion condition:
```

File: docs/auto-qc/iqms_intepretability.md [hunk 4]
```diff
@@ -0,0 +1,1288 @@
...
+```python
+metrics_sets = {f"set{i+1}": list(iqms_scaled.columns[i*10:i*10+9]) for i in range(6)}
+metrics_sets["set8"] = list(iqms_scaled.columns[60:])
+for metrics_set in metrics_sets.values():
+     sns.pairplot(iqms_scaled[metrics_set])
+```
+
+
+    
+![png](copy_interpretability-of_iqms_files/copy_interpretability-of_iqms_24_0.png)
```
Comment (oesteban) [inline docs/auto-qc/iqms_intepretability.md]: I'm not sure you need this, the notebook will be run on the data.

File: docs/auto-qc/iqms_intepretability.md [hunk 5]
```diff
@@ -0,0 +1,1288 @@
...
+3. bad quality.
+
+The neuroradiologist were asked to harmonize their ratings, hence the dataset contains only one score per images.
+
+
+```python
+# set path to the data
+path_data = os.path.abspath("../data/")
+# import IQMs
+iqms = pd.read_csv(os.path.join(path_data, "IQMs.tsv"), index_col="bids_name", sep="\t")
```
Comment (oesteban) [inline docs/auto-qc/iqms_intepretability.md]: I don't see `.../data/IQMs.tsv` uploaded anywhere...

**Review by oesteban — COMMENTED** [review]
Going in the right direction! Thanks for this!

File: docs/auto-qc/iqms_intepretability.md (hunk 4 above)
Comment (celprov) [reply docs/auto-qc/iqms_intepretability.md]: Let's see when we push the notebook.

**Comment by oesteban** [comment]
Okay, can you rebase on main? I just fixed the github action that was failing

**Comment by celprov** [comment]
> Okay, can you rebase on main? I just fixed the github action that was failing

Done!

**Pull request merged**